web-frame-error/index.html

```diff
@@ -0,0 +1,112 @@
+<!--
+  @license
+  Copyright (c) 2018-2020 THREEANGLE SOFTWARE SOLUTIONS SRL
+  Available under MIT license webFrame/LICENSE
+-->
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <meta content="ie=edge" http-equiv="X-UA-Compatible">
+  <title> Runtime Error!</title>
+  <base href="/">
+  <style>
+    body {
+      display: grid;
+      font-family: "-apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", system-ui, sans-serif;
```

Comment:
FYI: This font stack includes the native fonts for most operating systems.

Inspired from: https://design-system.protontech.ch/typography/